client/css/relatedProducts/ComparisonModal.scss

```diff
@@ -1,19 +1,19 @@
-$modal-width: 450px;
+$modal-width: 50%;
```

Comment:
is this the way to create a CSS variable? does it require importing JQuery?

Reply:
Yeah, I forget this file is SCSS.

Reply:
Yep. They can be useful. Kind of like self-documenting code. 
- https://sass-lang.com/documentation/variables